internal/chartverifier/checks/helm_test.go

```diff
@@ -180,7 +180,7 @@ func TestLongLineTemplate(t *testing.T) {
 	content, err := os.ReadFile("templates/test-template.yaml")
 	require.NoError(t, err)
 
-	images, err := getImagesFromContent(string(content))
+	images, err := getImagesFromContent(string(content)), nil
```

Comment:
Injecting `nil` here looks odd to me. This is because our return values changed in the signature?

Also, should we extend our test coverage to include an empty image value check?

Reply:
Correct. The function signature of the `getImagesFromContent()` function changed. So in an effort to avoid changing a ton of things, left that as a `nil` for now.